app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/view/shipping-information.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Not sure that it's the right way to change template for all delivery methods. We need to change it in case with In-Store Pickup selected only.

Reply:
@sidolov, fixed.